docs/source/ko/_toctree.yml

```diff
@@ -70,83 +71,114 @@
-  title: 태스크 가이드
-- sections:
-    - local: fast_tokenizers
-      title: 🤗 Tokenizers 라이브러리에서 토크나이저 사용하기
-    - local: multilingual
-      title: 다국어 모델 추론하기
+  - isExpanded: false
+    sections:
     - local: generation_strategies
       title: 텍스트 생성 전략 사용자 정의
-    - local: create_a_model
-      title: 모델별 API 사용하기
-    - local: custom_models
-      title: 사용자 정의 모델 공유하기
-    - local: sagemaker
-      title: Amazon SageMaker에서 학습 실행하기
-    - local: serialization
-      title: ONNX로 내보내기
-    - local: tflite
-      title: TFLite로 내보내기
-    - local: torchscript
-      title: TorchScript로 내보내기
-    - local: in_translation
-      title: (번역중) Benchmarks
-    - local: in_translation
-      title: (번역중) Notebooks with examples
-    - local: community
-      title: 커뮤니티 리소스
-    - local: custom_tools
-      title: 사용자 정의 도구와 프롬프트
-    - local: troubleshooting
-      title: 문제 해결
+    title: 생성
+  - isExpanded: false
+    sections:
+    - local: in_translation
+      title: (번역중) Image tasks with IDEFICS
+    - local: in_translation
+      title: (번역중) LLM prompting guide
+    title: (번역중) 프롬프팅
+  title: 태스크 가이드
+- sections:
+  - local: fast_tokenizers
+    title: 🤗 Tokenizers 라이브러리에서 토크나이저 사용하기
+  - local: multilingual
+    title: 다국어 모델 추론하기
+  - local: create_a_model
+    title: 모델별 API 사용하기
+  - local: custom_models
+    title: 사용자 정의 모델 공유하기
+  - local: in_translation
+    title: (번역중) Templates for chat models
+  - local: in_translation
+    title: (번역중) Trainer
+  - local: sagemaker
+    title: Amazon SageMaker에서 학습 실행하기
+  - local: serialization
+    title: ONNX로 내보내기
+  - local: tflite
+    title: TFLite로 내보내기
+  - local: torchscript
+    title: TorchScript로 내보내기
+  - local: in_translation
+    title: (번역중) Benchmarks
+  - local: in_translation
+    title: (번역중) Notebooks with examples
+  - local: community
+    title: 커뮤니티 리소스
+  - local: custom_tools
+    title: 사용자 정의 도구와 프롬프트
+  - local: troubleshooting
+    title: 문제 해결
+  - local: hf_quantizer
+    title: Contribute new quantization method
   title: (번역중) 개발자 가이드
 - sections:
-    - local: performance
-      title: 성능 및 확장성
+  - local: performance
+    title: 성능 및 확장성
+  - local: in_translation
+    title: (번역중) Quantization
+  - sections:
     - local: in_translation
       title: (번역중) Training on one GPU
     - local: perf_train_gpu_many
       title: 다중 GPU에서 훈련 진행하기
+    - local: in_translation
+      title: (번역중) Fully Sharded Data Parallel
+    - local: in_translation
+      title: (번역중) DeepSpeed
     - local: perf_train_cpu
       title: CPU에서 훈련
     - local: perf_train_cpu_many
       title: 다중 CPU에서 훈련하기
-    - local: in_translation
-      title: (번역중) Training on TPUs
     - local: perf_train_tpu_tf
       title: TensorFlow로 TPU에서 훈련하기
     - local: in_translation
-      title: (번역중) Training on Specialized Hardware
-    - local: perf_infer_cpu
-      title: CPU로 추론하기
-    - local: perf_infer_gpu_one
-      title: 하나의 GPU를 활용한 추론
-    - local: perf_infer_gpu_many
```

Comment:
One last thing, since we're removing the `perf_infer_gpu_many` [doc](https://github.com/huggingface/transformers/blob/main/docs/source/ko/perf_infer_gpu_many.md), it should also be removed from `docs/source/ko`.